src/main/java/medicaldoctor/LoginController.java

```diff
@@ -25,7 +36,15 @@ void buttonSubmitClick(ActionEvent event) throws Exception {
                 textUsername.getText(),
                 textPassword.getText());
         if (result == LoginResult.SUCCESS) {
-            System.out.println("good!");
+            System.out.println("right username");
```

Comment:
we don't need print out if the scene transition is implemented.

Reply:
This is there for me to troubleshoot.

Reply:
but if you are not done troubleshooting then you are not done implementing this work. am I misunderstanding when I say you have completed the goto additional password after logging in correctly?